resources/views/Homepage/HomepageCategory.twig

```diff
@@ -2,7 +2,7 @@
 
 {% block PartialHead %}
     {% embed getPartial('page-metadata') %}
-        {% block title '' %}
+        {% block title details.metaTitle | default('') %}
```

Comment:
Die Variable `details` musst du an dieser Stelle erst definieren.

```suggestion
        {% block title category.details[0].metaTitle | default('') %}
```

Reply:
Nach meinem Verständnis nicht unbedingt, da details in page-metadata definiert wird und dort anscheinend erst details.metaTitle als Expression geparsed wird.
Aber tatsächlich besser für das Verständnis.